projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-conditional-filter.component.html

```diff
@@ -1,5 +1,6 @@
 <ng-container *ngIf="esf.column">
     <div tabindex="0"
+        id='excelFilter'
```

Comment:
```suggestion
        id='esfConditionalFilter'
```

Reply:
@onlyexeption , do the same rename in `drop-down.component.ts` and `excel-style-conditional-filter.component.ts`

Reply:
@onlyexeption , this is also considered a bug and should be fixed for 15.1.x